apps/argus/src/keeper/hermes.rs

```diff
@@ -0,0 +1,413 @@
+/// Hermes Integration
```

Comment:
Since this is a client for another API I think keeper might not be the best module for it. You can have an `adaptor` module for Hermes.

For future: Pyth Native, Lazer, and now Argus are all using Hermes in Rust and each created their own client for it. We should make a Rust client for Hermes!